editor.planx.uk/package.json

```diff
@@ -10,7 +10,6 @@
     "@emotion/styled": "^11.11.0",
     "@mui/icons-material": "^5.15.2",
     "@mui/material": "^5.15.2",
-    "@mui/styles": "^5.15.2",
```

Comment:
this feels like a big one / nice milestone on our MUI migration to styled components :100:

Reply:
💯